### ISSUE TITLE
citation keys from BibText GUI picker all in capitals, different than Zotero default - and not working with ODT scane 

### ISSUE THREAD

**Issue by ymshulman** [retorquere/zotero-better-bibtex#826]
Hi all! I appreciate what you've created, as individuals and as a community, and would like to ask for some advice.

I've been working to get a workflow going using Scrivener to write, invoking the BibTex picker via a script to select citations and have them pasted in Scrivener in the regular Zotero scannable cite format, and using the Zotero plugin with LibreOffice to get everything sorted out. 

I'm using a script that has the following where it requests a citekey from BibTex:

```
else if zotRunning is "ready" then
	set theReference to do shell script "/usr/bin/curl 'http://localhost:23119/better-bibtex/cayw?format=scannable-cite' 2>/dev/null; exit 0"
	tell application "Scrivener"
		activate
		repeat until application "Scrivener" is frontmost
		end repeat
		tell application "System Events"
			keystroke theReference
		end tell

```

However, the citation that comes in is in a slightly different format than when I would select "copy as Scannable Cite" from within the Zotero menu. 

The Zotero would look like this: { | Zaccaro, 2014 | | |zu:951617:TCA6HSMD}
And the one I'm getting looks like this: { | ZACCARO< 2014 |  |  | ZU:951617:TCA6HSMD }

Using ODT scan and opening the resulting document in LibreOffice, only the first, default Zotero citations are getting converted properly. The second type, the ones that BibText creates, are not. 

Bottom line, everything seems to be in place with the scripts to invoke the BibTex GUI, having them paste into Zotero properly, running the ODT scan via LibreOffice, etc - but the actual cite keys are different. 

I apologize if I've used some wrong terminology! I'm pretty technically proficient, but do not know how to write scripts.

**Comment by retorquere** [comment]
There's not really a community, only me.

I'm not entirely sure where you would copy as Scannable Cite" from within the Zotero menu -- I can't find it -- but can you export the references using the `Scannable Cite` exporter to see if they are different than what you get from the picker?

**Comment by ymshulman** [comment]
Whoa. That's a lot of productivity that you're enabling, single-handedly. Kudos!

Export as scannable cite shows the same thing as the copied Scannable Cite from within the Zotero menu (it replaces the "copy as citation" and "copy as bibliography" options when choosing in the settings what I'm attaching in a screenshot here), and still different than what I'm getting with the picker. The picker seems to force everything into capitals, and the comma after the author name into a >

Here is what the output looks like from the export:

`{ | Zaccaro, 2014 | | |zu:951617:TCA6HSMD}{ | Chen, et al., 2014 | | |zu:951617:DDGK8NSN}{ | Nicolaides, et al., 2014 | | |zu:951617:WDCUZIZJ}`


Screenshots:
![screen shot 2017-11-28 at 5 09 44 pm](https://user-images.githubusercontent.com/10394892/33346884-2c863316-d45f-11e7-82a2-f5a42d98c8c8.png)
![screen shot 2017-11-28 at 5 10 13 pm](https://user-images.githubusercontent.com/10394892/33346885-2c958f3c-d45f-11e7-81f1-80f66f951fe1.png)

**Comment by ymshulman** [comment]
To follow up on this:

If I paste `http://localhost:23119/better-bibtex/cayw?format=scannable-cite`in my browser, I get a properly formatted cite marker. So it seems something about the rest of that line, `http://localhost:23119/better-bibtex/cayw?format=scannable-cite' 2>/dev/null; exit 0`is forcing the changes.

**Comment by retorquere** [comment]
And if you paste `/usr/bin/curl 'http://localhost:23119/better-bibtex/cayw?format=scannable-cite'` in a shell?

**Comment by ymshulman** [comment]
It comes up properly, in lowercase, as would be picked up by Zotero's ODT Scan.

**Comment by retorquere** [comment]
Then I'm afraid I can't help you -- I have no control over what happens after the results are fetched.

**Comment by ymshulman** [comment]
Okay - thanks for trying! I really appreciate what you're doing.

**Comment by retorquere** [comment]
My pleasure.
